doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -1701,35 +1722,34 @@ def find_payload_entries(cls, assembly_inspector: AssemblyInspector,
         :return: Map[payload_tag_name] -> PayloadEntry.
         """
 
-        members: Dict[str, PayloadEntry] = cls._find_initial_payload_entries(assembly_inspector, arch, dest_repo)
-        members = cls._replace_missing_payload_entries(members, arch)
-        rhcos_members, issues = cls._find_rhcos_payload_entries(assembly_inspector, arch)
+        members: Dict[str, PayloadEntry] = self._find_initial_payload_entries(assembly_inspector, arch, dest_repo)
+        members = self._replace_missing_payload_entries(members, arch)
+        rhcos_members, issues = self._find_rhcos_payload_entries(assembly_inspector, arch)
         members.update(rhcos_members)
         return members, issues
 
-    @staticmethod
-    def _find_initial_payload_entries(assembly_inspector: AssemblyInspector,
+    def _find_initial_payload_entries(self, assembly_inspector: AssemblyInspector,
                                       arch: str, dest_repo: str) -> Dict[str, PayloadEntry]:
 
         # Maps release payload tag name to the PayloadEntry for the image.
         members: Dict[str, Optional[PayloadEntry]] = dict()
-        for payload_tag, archive_inspector in \
-                PayloadGenerator.get_group_payload_tag_mapping(assembly_inspector, arch).items():
+        for payload_tag, image_inspector in \
+                self.get_group_payload_tag_mapping(assembly_inspector, arch).items():
```

Comment:
```suggestion
                self.payload_generator.get_group_payload_tag_mapping(assembly_inspector, arch).items():
```
? unless i'm missing something